src/update.ts

```diff
@@ -1,7 +1,35 @@
+import chalk from 'chalk'
 import * as updateNotifier from 'update-notifier'
 
 import * as pkg from '../package.json'
 
+const notifier = updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 })
```

Comment:
How about adding instantiating the notifier inside the `notify` function?

Reply:
Oh, my bad! Will do that asap!

Reply:
@tiagonapoli I had to stop programming for some time and am coming back and relearning everything, and I can't remember if exporting the constant in this case would be a good practice.

What do you think? I'm already doing a quick search to see if I can find something about it.

Reply:
In that case you wouldn't be exporting the `notifier` constant, it will just be declared inside the `notify` function:
```js
export default function notify() {
    const notifier = updateNotifier({ pkg, updateCheckInterval: 1 })  
    if (notifier.update && notifier.update.latest !== pkg.version) {
    const oldVersion = notifier.update.current
    ...
}
``` 

This way the `updateNotifier` is only instantiated when really needed (when the notify function is called) and it will be localized inside it will really be used. Of course this comes with the price that the `updateNotifier` may be instantiated several times, if the `notify` function is called many times, but this will not happen. Each `vtex someCommand` execution will call `notify` one time.

This change however is just me nitpicking, on a second though I think it's okay to leave the notifier where it's now :)

Reply:
I think you're right on this one, so I left it inside the function as you mentioned!